alloydb-jdbc-connector/src/main/java/com/google/cloud/alloydb/ConnectionConfig.java

```diff
@@ -17,21 +17,18 @@
 package com.google.cloud.alloydb;
 
 import com.google.cloud.alloydb.v1.InstanceName;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 import java.util.Properties;
-import java.util.stream.Collectors;
 
 public class ConnectionConfig {
```

Comment:
This class should be package-private, not public.

Reply:
Updated. Thanks for catching that.